features/addressing/managing_zones/editing_zone.feature

```diff
@@ -0,0 +1,48 @@
+@addressing
+Feature: Editing a zone
+    In order to change membership of areas
+    As an Administrator
+    I want to be able to edit a zone
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has zone "United States of America" with "Alabama" and "Arizona" provinces
+        And the store has zone "European Union" with "France" and "Germany" countries
+        And the store has zone "America" with "North America" and "South America" zones
+
+    @todo
+    Scenario: Removing province from zone
+        Given I want to modify "United States of America"
+        And I remove "Arizona" member
+        And I save my changes
+        Then I should be notified about successful edition
+        And the zone "United States of America" should have only "Alabama" province
```

Comment:
`And the zone "The United States of America" should have only the province "Alabama"`